official/eap/templates/eap-xp4-basic-s2i.json

```
@@ -0,0 +1,531 @@
+					"to": {
+						"kind": "ImageStreamTag",
+						"name": "${APPLICATION_NAME}-build-artifacts:latest"
+					}
+				},
+				"source": {
+					"contextDir": "${CONTEXT_DIR}",
+					"git": {
+						"ref": "${SOURCE_REPOSITORY_REF}",
+						"uri": "${SOURCE_REPOSITORY_URL}"
+					},
+					"type": "Git"
+				},
+				"strategy": {
+					"sourceStrategy": {
+						"env": [
+							{
+								"name": "MAVEN_MIRROR_URL",
+								"value": "${MAVEN_MIRROR_URL}"
+							},
+							{
+								"name": "MAVEN_ARGS_APPEND",
+								"value": "${MAVEN_ARGS_APPEND}"
+							},
+							{
+								"name": "GALLEON_PROVISION_LAYERS",
+								"value": "${GALLEON_PROVISION_LAYERS}"
+							},
+							{
+								"name": "GALLEON_PROVISION_DEFAULT_FAT_SERVER",
+								"value": "true"
+							},
+							{
+								"name": "ARTIFACT_DIR",
+								"value": "${ARTIFACT_DIR}"
+							}
+						],
+						"forcePull": true,
+						"from": {
+							"kind": "ImageStreamTag",
+							"name": "${EAP_IMAGE_NAME}",
+							"namespace": "${IMAGE_STREAM_NAMESPACE}"
+						},
+						"incremental": true
+					},
+					"type": "Source"
+				},
+				"triggers": [
+					{
+						"github": {
+							"secret": "${GITHUB_WEBHOOK_SECRET}"
+						},
+						"type": "GitHub"
+					},
+					{
+						"generic": {
+							"secret": "${GENERIC_WEBHOOK_SECRET}"
+						},
+						"type": "Generic"
+					},
+					{
+						"imageChange": {},
+						"type": "ImageChange"
+					},
+					{
+						"type": "ConfigChange"
+					}
+				]
+			}
+		},
+		{
+			"apiVersion": "v1",
+			"kind": "BuildConfig",
+			"metadata": {
+				"labels": {
+					"application": "${APPLICATION_NAME}"
+				},
+				"name": "${APPLICATION_NAME}"
+			},
+			"spec": {
+				"output": {
+					"to": {
+						"kind": "ImageStreamTag",
+						"name": "${APPLICATION_NAME}:latest"
+					}
+				},
+				"source": {
+					"dockerfile": "FROM ${EAP_RUNTIME_IMAGE_NAME}\nCOPY /server $JBOSS_HOME\nUSER root\nRUN chown -R jboss:root $JBOSS_HOME \u0026\u0026 chmod -R ug+rwX $JBOSS_HOME\nUSER jboss\nCMD $JBOSS_HOME/bin/openshift-launch.sh",
+					"images": [
+						{
+							"from": {
+								"kind": "ImageStreamTag",
+								"name": "${APPLICATION_NAME}-build-artifacts:latest"
+							},
+							"paths": [
+								{
+									"destinationDir": ".",
+									"sourcePath": "/s2i-output/server/"
+								}
+							]
+						}
+					]
+				},
+				"strategy": {
+					"dockerStrategy": {
+						"from": {
+							"kind": "ImageStreamTag",
+							"name": "${EAP_RUNTIME_IMAGE_NAME}",
+							"namespace": "${IMAGE_STREAM_NAMESPACE}"
+						},
+						"imageOptimizationPolicy": "SkipLayers"
+					},
+					"type": "Docker"
+				},
+				"triggers": [
+					{
+						"imageChange": {
+							"from": {
+								"kind": "ImageStreamTag",
+								"name": "${APPLICATION_NAME}-build-artifacts:latest"
+							}
+						},
+						"type": "ImageChange"
+					},
+					{
+						"type": "ConfigChange"
+					}
+				]
+			}
+		},
+		{
+			"apiVersion": "v1",
```

Please fix in your upstream:
```suggestion
			"apiVersion": "apps.openshift.io/v1",
```